src/controllers/scoreboard.js

```diff
@@ -6,4 +6,4 @@ export async function get(ctx) {
       ['score', 'DESC']],
     attributes: { exclude: [ 'email' ] }
   })
-}
```

Comment:
Why this change? Is editorconfig not installed? However go to your editor settings and add line at end of file

Reply:
However this is not for this pr, just in general